linux_boot_info/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "linux_boot_info"
```

Comment:
I think this should be `oak_linux_boot_info`. Or in fact even `oak_boot_info_linux`? (so that if  there were other boot infos in the future, the prefix would be the same).

Reply:
Actually this should be called `oak_linux_boot_params` as the main Linux C struct is called `boot_params`; that's where the name comes from, it's the Rust version of the Linux `boot_params`.